### DISCUSSION TITLE
RIS geometry

### DISCUSSION THREAD

**Discussion by Nobitoso99** [NVlabs/sionna#598] [Q&A]
Hello everyone,

I am currently working with the Sionna library and exploring the implementation of Reconfigurable Intelligent Surfaces (RIS). I am relatively new to this and would like to know if it is possible to model a 3D RIS using Sionna.

Thank you!

**Comment by jhoydis** [comment]
Hi @Nobitoso99,

The RIS model that we have implemented is described in detail here: https://nvlabs.github.io/sionna/em_primer.html#reconfigurable-intelligent-surfaces-ris

**Reply by llucastex** [reply]
Hi @jhoydis, can't get access to the link, it seems it was deleted. Do you have any updates on the Sionna x RIS integration?

**Comment by jhoydis** [comment]
Hi @llucastex,

The link has changed https://jhoydis.github.io/sionna-0.19.2-doc/em_primer.html#reconfigurable-intelligent-surfaces-ris
